metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -19,11 +18,13 @@
  * The abstract base class for all scheduled reporters (i.e., reporters which process a registry's
  * metrics periodically).
  *
+ * Since {@link Reporter} extends {@link java.io.Closeable} this class also satisfies that interface.
+ *
  * @see ConsoleReporter
  * @see CsvReporter
  * @see Slf4jReporter
  */
-public abstract class ScheduledReporter implements Closeable, Reporter {
+public abstract class ScheduledReporter implements Reporter {
```

Comment:
Please don't remove `Closeable` from `ScheduledReporter`, because the change is binary incompatible. That means we will force all 3rd reporters to upgrade to the latest version of `metrics-core`. I think it should be fine to just add `Closeable` to `Reporter`. It should be backward-compatible because `ScheduledReporter` and `JmxReporter` implement it already.